drivers/platform/chibios/chibios.mk

```diff
@@ -0,0 +1,29 @@
+#linking no-os driver 
+
+#no-os headers
+NOOS_INC :=	$(NOOS)/include/
+
+#no-os API
+NOOS_SRC :=	$(NOOS)/drivers/api/no_os_spi.c \
+			$(NOOS)/drivers/api/no_os_gpio.c \
+			$(NOOS)/drivers/api/no_os_i2c.c
+
+#linking platform
+NOOS_INC +=	$(NOOS)/drivers/platform/chibios/
+
+NOOS_SRC +=	$(NOOS)/drivers/platform/chibios/chibios_spi.c \
+		 	$(NOOS)/drivers/platform/chibios/chibios_i2c.c \
+ 	   	    $(NOOS)/drivers/platform/chibios/chibios_gpio.c \
+			$(NOOS)/drivers/platform/chibios/chibios_delay.c \
+			$(NOOS)/drivers/platform/chibios/chibios_alloc.c 
+				
+#link utils
+NOOS_SRC +=	$(NOOS)/util/no_os_util.c
+
+#linking drivers
+NOOS_INC +=	$(NOOS)/drivers/accel/adxl355/
+NOOS_SRC +=	$(NOOS)/drivers/accel/adxl355/adxl355.c
```

Comment:
Was this added for testing purposes?

Reply:
I understand. I think it would be a good idea to follow the same approach we do for the other platforms. If you want to use adxl355 driver as an example you would need to add support in projects/eval-adxl355-pmdz. Would that make sense in this case?

Reply:
Yes, it would. Working on it

Reply:
Going to be solved in an other PR